CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 2.0.0-rc.4 (2020-09-23):
```

Comment:
We're ready to go to `2.0.0`, right?

Reply:
Yes please, I was just following the pattern, when you are ready to go to 2.0.0 go ahead and change.